<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-2</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="uplabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>16</pointsize>
           <weight>9</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font: 75 16pt &quot;Arial&quot;;
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>学生信息管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="button1">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>999999</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(255, 255, 255)</string>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rbutton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-image: url(:/icon/delete.png);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rbutton3">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
border-image: url(:/icon/group.png)</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rbutton2">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
border-image: url(:/icon/add.png)</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rbutton4">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);
border-image: url(:/icon/edit.png)</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>700</width>
         <height>500</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>学生数据</string>
        </attribute>
        <widget class="QTableWidget" name="tableWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>691</width>
           <height>531</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image:url(:/icon/databasebackground3.webp);

color:rgb(119, 255, 217)；</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>计算</string>
        </attribute>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-10</y>
           <width>691</width>
           <height>541</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-image: url(:/icon/avgbackground.webp);</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>80</y>
            <width>521</width>
            <height>221</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1">
            <widget class="QLabel" name="label1">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border:1px solid;
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>平均数</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label4">
             <property name="font">
              <font>
               <pointsize>18</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border:1px solid;</string>
             </property>
             <property name="text">
              <string>得到的平均数</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="comboBox_2">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_2">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>点击获得平均值</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>查询</string>
        </attribute>
        <widget class="QWidget" name="widget" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>691</width>
           <height>521</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>600</width>
           <height>400</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>700</width>
           <height>9999999</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
background-image: url(:/icon/searchbackground2.webp)</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>50</y>
            <width>171</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>120</y>
            <width>611</width>
            <height>301</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="2" column="0">
            <widget class="QTableWidget" name="tableWidget_2">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="lineEdit_2">
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid  black;
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>请输入要查询的内容</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>60</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>40</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>点击查询</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../assets/takeatry.qrc"/>
 </resources>
 <connections/>
</ui>
